tensorflow/python/ops/array_ops.py

```diff
@@ -1577,6 +1612,39 @@ def sparse_mask(a, mask_indices, name=None):
 
 @tf_export("unique")
 def unique(x, out_idx=dtypes.int32, name=None):
+  """Finds unique elements in a 1-D tensor.
+
+  This operation returns a tensor `y` containing all of the unique elements of `x`
+  sorted in the same order that they occur in `x`. This operation also returns a
+  tensor `idx` the same size as `x` that contains the index of each value of `x` in
+  the unique output `y`. In other words:
+
+
+    y[idx[i]] = x[i] for i in [0, 1,...,rank(x) - 1]
+
+  Example usage:
+
+  >>> x = tf.constant([1, 1, 2, 4, 4, 4, 7, 8, 8])
+  >>> y, idx = unique(x)
+  >>> y
+  <tf.Tensor: id=8, shape=(5,), dtype=int32, numpy=
+  ... array([1, 2, 4, 7, 8], dtype=int32)>
```

Comment:
I don't think you need `...` in the result.

`...` means a different think in the output. See https://docs.python.org/3/library/doctest.html#doctest.ELLIPSIS

Reply:
Gotcha! Thanks, @yashk2810.